modules/dnn/test/test_caffe_importer.cpp

```diff
@@ -47,6 +47,21 @@
 
 namespace opencv_test { namespace {
 
+CV_ENUM(DNNTarget, DNN_TARGET_CPU, DNN_TARGET_OPENCL)
+static testing::internal::ParamGenerator<DNNTarget> availableBackends()
```

Comment:
> DNNTarget
> availableBackends

Looks like there is some mess with target/backend terminology.

Reply:
Thanks you! Replaced to `availableDnnTargets`.